### PR TITLE
PLF-8694 Move gamification global pages to global site

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/global/navigation.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<node-navigation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                 xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_6 http://www.gatein.org/xml/ns/gatein_objects_1_6"
+                 xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_6">
+  <priority>1</priority>
+  <page-nodes>
+    <node>
+      <name>gamification-earn-points</name>
+      <label>How to earn point</label>
```
Comment (rdenarie) [inline extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/global/navigation.xml]: Should we add i18n ?

File: extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/global/navigation.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      <visibility>SYSTEM</visibility>
+      <page-reference>portal::global::gameficationhelp</page-reference>
+    </node>
+
+    <node>
+      <name>achievements</name>
+      <label>Gamification Achievements</label>
```
Comment (rdenarie) [inline extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/global/navigation.xml]: Should we add i18n ?

File: extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/intranet/navigation.xml [hunk 3]
```diff
@@ -1,31 +0,0 @@
...
-    <node>
-      <name>achievements</name>
-      <label>Gamification Achievements</label>
-      <visibility>SYSTEM</visibility>
-      <page-reference>portal::intranet::achievements</page-reference>
-    </node>
-  </page-nodes>
-  <page-nodes>
-    <node>
-      <name>achievements/mybadges</name>
```
Comment (rdenarie) [inline extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/intranet/navigation.xml]: where is this page in global navigation ?

File: extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/global/navigation.xml (hunk 1 above)
Comment (boubaker) [reply extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/global/navigation.xml]: Yep, but as discussed, I kept it as it was defined for now. To enhance later (Crowdin jobs update, more tests...)

File: extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/global/navigation.xml (hunk 2 above)
Comment (boubaker) [reply extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/global/navigation.xml]: Yep, but as discussed, I kept it as it was defined for now. To enhance later (Crowdin jobs update, more tests...)

File: extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/intranet/navigation.xml (hunk 3 above)
Comment (boubaker) [reply extension/src/main/webapp/WEB-INF/conf/gamification/portal/portal/intranet/navigation.xml]: This page URI is useless in fact.

**Pull request merged**